pipelines/README.md

```diff
@@ -1,163 +1,129 @@
+**Note:** API keys and locally uploaded images are not exported for privacy and security.
+If you make use of such items you may want to create a tutorial video and share that with
+your submission in step 4, part 4.
 
-## Visual Blocks Gallery
 
-We highlight a set of community-contributed pipelines of Visual Blocks below:
+### Step 3: Preparing your pipeline for submission to the community
 
-### Interactive Graphics
+In order for your pipeline to be accepted you must follow these rules:
 
-*   [3D Photo](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fgraphics%2F3dphoto_portrait_depth.json)
-    with
-    [Portrait Depth API](https://blog.tensorflow.org/2022/05/portrait-depth-api-turning-single-image.html?linkId=8063793)
+1. Create a new folder for your custom pipeline within this repositories'
+   `pipelines` folder
 
-    [![3D Effects with Portrait Depth API in Visual Blocks](graphics/3dphoto_portrait_depth.gif)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fgraphics%2F3dphoto_portrait_depth.json)
+2. Ensure the name of this new folder is the name of your pipeline. For example
+   if you decide to call your pipeline "Cat on table detector" then name your
+   folder  `cat_on_table_detector` all lowercase, with underscores between words.
 
-*   [AR Sticker Effect](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fwebcam%2Far_sticker_effect.json)
-    with
-    [Mediapipe Face Landmark Detection](https://developers.google.com/mediapipe/solutions/vision/face_landmarker)
+3. Inside this folder you have created ensure you add the following files:
 
-    [![AR Sticker Effect in Visual Blocks](webcam/ar_sticker_effect.gif)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fwebcam%2Far_sticker_effect.json)
+   a) **name_of_pipeline.json** (*required*) The JSON file you downloaded when you
+   exported your pipeline from visual blocks. Be sure to name it in the same
+   way as the folder, for our example above it would be `cat_on_table_detector.json`.
+   
+   b) **name_of_pipeline.jpg** (*required*) A high resolution JPG image that is 1280x720
+   pixels in size (16:9 ratio) showing a key snapshot of the Visual Blocks pipeline
+   you created in action. This will be shown in search results on the visual blocks
+   community pipelines page.
+   
+   c) **name_of_pipeline.mp4** (*optional*) If you are feeling creative, you can make a
+   short up to 15 seconds max, MP4 video (with no sound) showing the pipeline in action. If
+   you do this, we shall use this once we support it, to enhance your search result with a looped video of   
```

Comment:
I see a line break in the rendered result: https://screenshot.googleplex.com/9xTkR9GJEF6gwXx